docs/getting_started.rst

```diff
@@ -175,6 +175,10 @@ Using any text editor to edit the Dockerfile and paste the following:
 .. literalinclude:: resources/Dockerfile
     :language: dockerfile
 
+.. note::
+
+    For nvflare version 2.3 set PYTORCH_IMAGE=nvcr.io/nvidia/pytorch:23.02-py3
+
```

Comment:
Do we need this? Do we want to add this to all the future versions?

Reply:
Unfortunately, for grpc version compatibility we do need to provide some way to let users know what is supported, so an alternative would be some sort of table that lists what is compatible

Reply:
For now it is just these two versions, but in the future it's a good idea to add a table